test/e2e/setup.go

```diff
@@ -50,9 +49,8 @@ import (
 const seleniumContainerName = "selenium-edge-standalone"
 
 type clientSet struct {
-	OpenshiftClustersv20200430 redhatopenshift20200430.OpenShiftClustersClient
-	Operationsv20200430        redhatopenshift20200430.OperationsClient
-	OpenshiftClustersv20220401 redhatopenshift20220401.OpenShiftClustersClient
```

Comment:
We would reintroduce different versions back in to ensure specific APIs had specific fields in the future correct?

Reply:
Maybe? I guess if we want to have e2e tests for future/preview APIs. I could name this as "OpenshiftClustersDefault" or something if we wanted to be able to add more clients later.

I think @m1kola might also have some input here as I believe he's been working on our tests in this area.

Reply:
Agreed @m1kola, those would both be great enhancements to the e2e tests.. both testing all supported API versions, as well as some way to flag certain tests as only applying to specific versions.

Are you able to take a look for that historical card? Happy to get it pulled over to JIRA if you can find it. Otherwise we can get a new fresh one in JIRA cut.